Makefile.morello

```diff
@@ -2,7 +2,7 @@ CC=$(HOME)/cheri/output/morello-sdk/bin/clang
 CFORMAT=$(HOME)/cheri/output/morello-sdk/bin/clang-format
 CXX=$(HOME)/cheri/output/morello-sdk/bin/clang++
 CFLAGS=-fuse-ld=lld --config cheribsd-morello-purecap.cfg
-SSHPORT=10085
+SSHPORT=12351
```

Comment:
I don't think this should have changed?

Reply:
It doesn't matter. It's my port. In general, this is pretty random. 12351, instead, is the port I have got assigned by FVP (config didn't work in this case). Anyway, I'll put it back to 10085 (which is a personal mnemonic I don't think anyone cares :))

Reply:
I suspect this should be `SSHPORT ?= ...` which allows the user to override it in the environment.